### PR TITLE
ci(release): publish service images to docker-hub

### PR DESCRIPTION
**Pull request by kelvin-olaiya**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -37,11 +49,56 @@ jobs:
...
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - name: Build and push
+        uses: docker/build-push-action@v6.3.0
+        with:
+          context: ${{ matrix.image }}
+          platform: linux/amd64,linux/arm64
+          file: ./Dockerfile
+          push: true
+          tags: |
+            letsdothisshared/revue-${{ steps.compute-next-version.outputs.next-version }}
```
Comment (Mala1180) [inline .github/workflows/release.yml:103]:
```suggestion
            letsdothisshared/revue-${{ matrix.image }}:${{ steps.compute-next-version.outputs.next-version }}
```

**Review by Mala1180 — CHANGES_REQUESTED** [review]
Nice!

File: .github/workflows/release.yml (hunk 1 above)
Comment (kelvin-olaiya) [reply .github/workflows/release.yml:103]: aaahh yess, silly me :)
Thks!

**Comment by kelvin-olaiya** [comment]
This PR will be sqaushed

**Pull request merged**